### DISCUSSION TITLE
Network models

### DISCUSSION THREAD

**Discussion by Haaroon** [Pometry/Raphtory#827] [Bounty Board]
## Network models

* Configuration model (also for the randomisation of an empirical graph)
* Erdos-Renyi $G(N,p)$
* Erdos-Renyi $G(N,M)$ :grapes: 
* Preferential attachment model
* Random geometric graphs
* Small-World Model

**Comment by phdavis1027** [comment]
Working on G(N,M) erdos-renyi, wish me luck

**Reply by phdavis1027** [reply]
Actually, it seems that G(N, p) has already been implemented. Is G(N, M) still wanted?
